src/refresh_status.c

```diff
@@ -35,6 +35,24 @@ void on_hide_clicked(GtkButton *button, RefreshState *state) {
   refresh_state_free(state);
 }
 
+void on_main_window_size_allocate(GtkApplicationWindow *self,
+                                  GtkAllocation *allocation,
+                                  RefreshState *state) {
+  gboolean modified = FALSE;
+  if (allocation->width > state->width) {
+    modified = TRUE;
+    state->width = allocation->width;
+  }
+  if (allocation->height > state->height) {
+    modified = TRUE;
+    state->height = allocation->height;
+  }
+  if (modified) {
+    printf("Change size\n");
```

Comment:
Let's drop the print statement.

Reply:
Oops... yeah, my fault... debugging notes O:-)